File: 10-minimum-maximum-number/main.cpp
/*
  Ernest Izdebski, 2022-09-07

  A program that asks the user to enter two numbers, and uses the conditional operator to determine which number is smaller and which is the larger, or are they equal.
*/

#include <iostream>
using namespace std;

int main() {
  // define number variables
  // I use doubles because the author *might* choose a decimal number
  double n_0;
  double n_1;

  // populate variables
  cout << "Input 1st number: ";
  cin >> n_0;
  cout << "Input 2nd number: ";
  cin >> n_1;

  // perform conditional logic
  if (n_0 < n_1) {
    cout << n_0 << " is less than " << n_1 << endl;
  } else if (n_0 == n_1) {
    cout << n_0 << " is equal to " << n_1 << endl;
  } else if (n_0 > n_1) {
    cout << n_0 << " is greater than " << n_1 << endl;
  }
}

File: 10-minimum-maximum-number/src/main.rs
mod ernie;

fn main() {
    let x: f32 = ernie::input("input 1st number: ");
    let y: f32 = ernie::input("input 2nd number: ");
    println!();

    if x < y {
        println!("{} is less than {}", x, y);
    } else if x > y {
        println!("{} is greater than {}", x, y);
    } else if x == y {
        println!("{} is equal to {}", x, y);
    } else {
        println!("either '{}' and/or '{}' is NaN", x, y);
    }
}
